cmd/kubelet/app/options/options.go

```diff
@@ -94,7 +94,7 @@ func NewKubeletServer() *KubeletServer {
 			MaxContainerCount:           100,
```

Comment:
Yes, we should probably increase this. Unfortunately, our container GC implementation is far from perfect. This literally means *when* the GC will kick in. By changing this to 200, the GC won't start until there are 200 dead containers, even if you only run 3 pods on the node. /cc @dchen1107 @vishh